docs/code-style-guide.md

```diff
@@ -75,6 +75,22 @@ If you want to know more about our coding style you can take a look at our [clan
   const float SCALE_FACTOR = 2.15;
   float distance = catch_distance * SCALE_FACTOR;
   ```
+* Avoid initializing multiple variables on the same line.
+  ```cpp
+  // Incorrect
+  int x, y, z = 0;
+
+  // Correct and equivalent to the above
+  int x;
+  int y;
+  int z = 0;
+
+  // However, the author may have intended the following
+  // or a code reader may have assumed the following
```

Comment:
@garethellis0 Here I explain why we "Avoid initializing multiple variables on the same line." I show by example